examples/fib.c

```diff
@@ -61,7 +61,15 @@ static int js_fib_init(JSContext *ctx, JSModuleDef *m)
 #define JS_INIT_MODULE js_init_module_fib
 #endif
 
-JSModuleDef *JS_INIT_MODULE(JSContext *ctx, const char *module_name)
+#ifndef JS_EXTERN
+#ifdef _WIN32
+#define JS_EXTERN __declspec(dllexport)
+#else
+#define JS_EXTERN
+#endif
+#endif
```

Comment:
JS_EXTERN is always defined, right? Why is this necessary?

Reply:
Indeed, let's export this properly in quickjs.h @zeromake

Reply:
@saghul 
I tried `#include "../quickjs.h"` use `JS_EXTERN`
``` sh
/Users/zero/project/quickjs-ng/examples/fib.c:72:1: error: unknown type name 'JS_EXTERN'
JS_EXTERN JSModuleDef *JS_INIT_MODULE(JSContext *ctx, const char *module_name)
^
/Users/zero/project/quickjs-ng/examples/fib.c:72:22: error: expected ';' after top level declarator
JS_EXTERN JSModuleDef *JS_INIT_MODULE(JSContext *ctx, const char *module_name)
```
on quickjs.h has `JS_EXTERN` ……

Reply:
quickjs.h latest is `#undef JS_EXTERN` ……
https://github.com/quickjs-ng/quickjs/blob/5bf35450cd0e03b9a6576f3d9db22b1e1ea20548/quickjs.h#L1026

Reply:
@saghul @bnoordhuis 
https://github.com/quickjs-ng/quickjs/pull/490/commits/a9239349a095ac3b074ab0cf2ba5f897648f1f25
del `#undef JS_EXTERN`